### PR TITLE
Expose preferredDisplayName property in participant.

### PR DESCRIPTION
**Pull request by zihzhan-msft**

### CODE REVIEW

File: changelog/Microsoft.Skype.Calling.json [hunk 1]
```diff
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "840dd994-b26b-48a2-8d7c-1b52ea86a45c",
+          "ApiChange": "Property",
+          "ChangedApiName": "preferredDisplayName",
+          "ChangeType": "Addition",
+          "Description": "Added the `preferredDisplayName` property to the [participant](https://learn.microsoft.com/en-us/graph/api/resources/participant?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.Skype.Calling.json:10]:
```suggestion
          "Description": "Added the **preferredDisplayName** property to the [participant](https://learn.microsoft.com/en-us/graph/api/resources/participant?view=graph-rest-beta) resource.",
```

**Review by Danielabom — APPROVED** [review]
PR approved.

**Comment by Danielabom** [comment]
@zihzhan-msft , please add the "ready to merge" label as soon as you are ready to publish this content. Thanks!

**Comment by zihzhan-msft** [comment]
Thank you @Danielabom for reviewing this PR

**Pull request merged**